### ISSUE TITLE
[Bug]: Error in Model Plot

### ISSUE THREAD

**Issue by IsabelVarela** [jasp-stats/jasp-issues#1570]
### JASP Version

0.16

### JASP Module

jaspFactor

### What analysis are you seeing the problem on?

CFA

### What OS are you seeing the problem on?

Windows 10

### Bug Description

All the processes run successfully, but when I asked for the Model plot this error occur 

```
Error in bars[[node]] <- pnorm(GroupThresh$est[GroupThresh$lhs = GroupVars$name[node]]): attempt to select more than one element in integerOneIndex

Stack trace
tryCatchOne(expr, names, parentenv, handlers[[1]])

doTryCatch(return(expr), name, parentenv, handler)

withCallingHandlers(expr = analysis(jaspResults = jaspResults, dataset = dataset, options = options), error = .addStackTrace)

analysis(jaspResults = jaspResults, dataset = dataset, options = options)

ConfirmatoryFactorAnalysis(...)

.cfaPlotPath(jaspResults, options, cfaResult)

jaspBase:::.suppressGrDevice(pathplot <- semPlot::semPaths(object = .cfaLavToPlotObj(cfaResult[['lav']], options), DoNotPlot = TRUE, ask = FALSE, layout = 'tree', intercepts = options$plotmeans, whatLabels = if (!options$plotpars) 'name' else if (options$plotstd) 'std' else 'par', mar = ifelse(rep(is.null(options$secondOrder), 4), c(12, 3, 12, 3), c(6, 3, 6, 3)), edge.color = 'black', color = list(lat = '#EAEAEA', man = '#EAEAEA', int = '#FFFFFF'), border.width = 1.5, edge.label.cex = 0.9, lty = 2,
    title = FALSE))

eval(plotFunc, parent.frame())

eval(plotFunc, parent.frame())

semPlot::semPaths(object = .cfaLavToPlotObj(cfaResult[['lav']], options), DoNotPlot = TRUE, ask = FALSE, layout = 'tree', intercepts = options$plotmeans, whatLabels = if (!options$plotpars) 'name' else if (options$plotstd) 'std' else 'par', mar = ifelse(rep(is.null(options$secondOrder), 4), c(12, 3, 12, 3), c(6, 3, 6, 3)), edge.color = 'black', color = list(lat = '#EAEAEA', man = '#EAEAEA', int = '#FFFFFF'), border.width = 1.5, edge.label.cex = 0.9, lty = 2, title = FALSE)
```

### Expected Behaviour

I need the Model plot

### Steps to Reproduce

1. import data from spss
2. define the latent variables (4 factors)
3. Ask for factor variances
4. Ask fit measures, R-squared, the 2 matrices of covariances, modifications indices by default (3.84)
5. Error 95%, method standard, DWLS and all standardization
...

### Final Checklist

- [X] I have included a screenshot showcasing the issue, if any.
- [X] I have included a JASP file that causes the crash/bug, if any.
- [X] I have accurately described the bug, and steps to reproduce it.

**Comment by boutinb** [comment]
Hi @IsabelVarela Would it be possible to send a JASP file that shows this problem? To send a JASP file, you need first to zip it, and then drag&drop it in a new comment of this issue in GitHub.

**Comment by Kucharssim** [comment]
Dear @IsabelVarela ,

thank you for submitting this issue! We were already made aware of it previously (#1548) and fixed it since then (jasp-stats/jaspFactor#69). The fixed version of the analysis will be available in the next JASP release (which should be out soon).

I am closing this issue. Please feel free to reopen the issue in case the problem persists in the future versions.
